README.md

```diff
@@ -22,24 +36,24 @@ For historical reasons this repository is called "visualfsharp" and currently al
 |dev15.9|[![Build Status](https://dnceng.visualstudio.com/_apis/public/build/definitions/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/106/badge?branchname=dev15.9)](https://dnceng.visualstudio.com/public/public%20Team/_build?definitionId=106&_a=history)|
 |dev16.0|[![Build Status](https://dnceng.visualstudio.com/_apis/public/build/definitions/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/106/badge?branchname=dev16.0)](https://dnceng.visualstudio.com/public/public%20Team/_build?definitionId=106&_a=history)|
 
-## Help improve the Quality of the Tools by Using the Nightly Releases of Visual F# Tools
-To setup Visual Studio to use the latest nightly releases of the Visual F# Tools:
-https://blogs.msdn.microsoft.com/dotnet/2017/03/14/announcing-nightly-releases-for-the-visual-f-tools/
+## Using nightly releases in Visual Studio
 
+You can use the latest `master` build of the F# compiler and tools for Visual Studio via our nightly releases if you are a Visual Studio users. See details on setup here:
 
-## Contributing
+https://blogs.msdn.microsoft.com/dotnet/2017/03/14/announcing-nightly-releases-for-the-visual-f-tools/
 
-See [DEVGUIDE.md](DEVGUIDE.md) and [TESTGUIDE.md](TESTGUIDE.md) for details on build, development, and testing.
+### Even more nightly than the nightly
 
-See [CONTRIBUTING.md](CONTRIBUTING.md) for general guidelines on the contribution process, also [how we label issues and PRs](https://github.com/dotnet/roslyn/wiki/Labels-used-for-issues)
+Alternatively, if you _really_ want to live on the bleeding edge, you can set up a nightly feed for the Visual Studio preview releases, which use the latest commit in the preview branch. To do so, follow the same instructions as the above blog post, but instead with these links:
 
-To contribute to the F# ecosystem more generally see the F# Software Foundation's [Community Projects](http://fsharp.org/community/projects/) pages.
+* Set your feed to the [preview feed](https://dotnet.myget.org/F/fsharp-preview/vsix)
```

Comment:
Maybe just put the links in verbatim? You don't actually want to visit this link in your web browser.